keps/sig-network/2091-cluster-network-policy/README.md

```diff
@@ -240,68 +240,58 @@ Now suppose the client in each Namespace initiates traffic towards x/server.
 
 ### User Stories
 
-![Alt text](user_story_diagram.png?raw=true "User Story Diagram")
+Note: This KEP will focus on East-West traffic, cluster internal, user stories and 
+not address North-South traffic, cluster external, use cases, which will be 
+solved in a follow-up proposal.
 
-#### Story 1: Deny traffic from certain sources
+#### Story 1: Deny traffic at a cluster level 
 
-As a cluster admin, I want to explicitly deny traffic from certain source IPs
-that I know to be bad.
+As a cluster admin, I want to explicitly isolate certain pod(s) and(or) 
+Namespace(s) from all other cluster internal traffic. 
 
-Although this is a valid use case from an administrators' perspective, this KEP
-will not address this use case as we solely plan to focus developing an API
-which takes care of cluster internal traffic. All cluster external traffic use
-cases will be solved as a follow-up proposal.
+![Alt text](explicit_isolation.png?raw=true "Explicit Deny")
 
-#### Story 2: Ensure traffic goes through ingress/egress gateways
+#### Story 2: Allow traffic at a cluster level
 
-As a cluster admin, I want to ensure that all traffic coming into (going out of)
-my cluster always goes through my ingress (egress) gateway.
+As a cluster admin, I want to explicitly allow traffic to certain pods(s) 
+and(or) Namespace(s) from all other cluster internal traffic. 
```

Comment:
Just like the prior figure states that it is a non-overridable DENY, the figure and the text for Story 2 should state that this is a non-overridable ALLOW.

Reply:
Agreed, fixing